.github/workflows/docs.yml

```diff
@@ -0,0 +1,25 @@
+name: "Docs"
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-docs:
+    name: docs
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.8"
+    - name: Install nox
+      run: |
+        python -m pip install --upgrade setuptools pip wheel
+        python -m pip install nox
+    - name: Run docs
+      run: |
+        nox -s docs
```

Comment:
Since we're removing `docs-presubmit` from Kokoro, we actually need `nox -s docfx` as well for Cloud RAD.

`docs` used to run with all the unit and system tests.

Reply:
Fixed